docs/onnx-conversion.md

```diff
@@ -19,39 +19,59 @@ pip install torch transformers onnx onnxruntime onnxoptimizer
 
 versions tested:
 ```bash
-onnx                         1.16.1
+onnx                         1.17.0
 onnxoptimizer                0.3.13
-onnxruntime                  1.18.1
+onnxruntime                  1.20.1
 ```
 
 ## Converting from PyTorch models to ONNX model
-The following sections will describe how to convert SPLADE++ model to ONNX model. The steps are as follows:  
+The following sections will describe how to convert a Transformer-based model to ONNX model. We use the SPLADE++ model as an example. The steps are as follows:  
```

Comment:
We use lowercase t for transformers.

Reply:
We also use the convention of each sentence on its own line.
This makes diffs easier to read.